quantization_troubleshooting.md

```diff
@@ -0,0 +1,153 @@
+There are many buttons to push and knobs to tune in MCT in case the default values fail to deliver.
+The following are the usual suspects.
+
+#### Outlier Removal
+In some models, you may need to manually limit the activation thresholds using the `z_threshold` attribute in the `QuantizationConfig` in `CoreConfig`.
+
+**Solution:**
+Set a value to the `z_threshold`. Typical value range is between 5.0 and 40.0:
+```python
+core_config = mct.core.CoreConfig(mct.core.QuantizationConfig(z_threshold=17.0))
+quantizted_model, quantization_info = mct.ptq.keras_post_training_quantization(model, representative_dataset,
+                                                                               core_config=core_config)
+```
+
+#### Shift Negative Activation
+Some activation functions are harder to quantize than others (e.g. the **swish**, **Leaky ReLU**, etc.).
+These activation layers have negative values that use only a small part of the dynamic range.
+In these cases, shifting negative values to positive values can improve quantization error (saving an extra 1bit).
+
+**Solution:** This exactly what the `shift_negative_activation_correction` flag does.
+```python
+core_config = mct.core.CoreConfig(mct.core.QuantizationConfig(shift_negative_activation_correction=True))
+quantizted_model, quantization_info = mct.ptq.keras_post_training_quantization(model, representative_dataset,
+                                                                               core_config=core_config)
+```
+*note:* after activating this flag, you have a few more tweaks to its operation that you can control with the
+`shift_negative_ratio`, `shift_negative_threshold_recalculation` & `shift_negative_params_searc` flags.
+Read all about them in the [quantization config](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/core/common/quantization/quantization_config.py#L83) 
+
+
+### Bias Correction
+MCT applies bias correction by default to overcome induced bias shift caused by weights quantization.
+The applied correction is an estimation of the bias shift that is computed based on (among other things) the collected statistical data generated with the representative dataset.
+Therefore, the effect of the bias correction is sensitive to the distribution and size of the provided representative dataset.
+
+**Solution**: Verify the bias correction causes a degradation in accuracy by disabling it:
+```python
+core_config = mct.core.CoreConfig(mct.core.QuantizationConfig(weights_bias_correction=False))
+quantizted_model, quantization_info = mct.ptq.keras_post_training_quantization(model, representative_dataset,
+                                                                               core_config=core_config)
+```
+If you can increase your representative dataset and its distribution, it may restore accuracy.
+If you don't have an option to increase or diversify your representative dataset, disable the bias correction.
+
+
+## Common Issues
```

Comment:
I think that the "common issues" section needs to be removed and split:
- the point regarding concatenation can be a "tip"
- the point regarding torch.fx need to be moved to the FAQ and slightly rephrased (refer to it as "how to handle graph reading errors" or something like that)